app/helpers/brexit_checker_helper.rb

```diff
@@ -8,14 +8,16 @@ def format_criteria_list(criteria)
   def format_action_audiences(actions)
     action_groups = actions.group_by(&:audience)
 
-    action_groups.map do |key, action_group|
+    action_groups = action_groups.map do |key, action_group|
```

Comment:
could you call this something else so that we know action_groups has changed from hash to array